core/services/gateway/connector/connector.go

```diff
@@ -194,9 +216,6 @@ func (c *gatewayConnector) reconnectLoop(gatewayState *gatewayState) {
 func (c *gatewayConnector) Start(ctx context.Context) error {
 	return c.StartOnce("GatewayConnector", func() error {
 		c.lggr.Info("starting gateway connector")
-		if err := c.handler.Start(ctx); err != nil {
```

Comment:
I don't see any tests that had to change because the Start and Close were removed.  How is it tested that the all the handlers are started and stopped?

Reply:
These two lines were removed from connector_test.go:
`
handler.On("Start", mock.Anything).Return(nil)
handler.On("Close").Return(nil)
`